frontend/src/components/AdminQuiz/AdminQuizBody.tsx

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
-import TableBody from '../Table/TableBody';
-import TableBodyRow from '../Table/TableBodyRow';
-import TableBodyRowData from '../Table/TableBodyRowData';
-import AdminQuizAction from './AdminQuizAction';
-
-import { Quiz } from './types';
+import { Quiz, AdminQuizAction } from '.';
+import { TableBodyRow, TableBodyRowData, TableBody } from '../Table';
 
 interface AdminQuizBodyProps {
   records: Quiz[];
 }
 
-const AdminQuizBody = ({ records }: AdminQuizBodyProps): JSX.Element => {
+export const AdminQuizBody = ({ records }: AdminQuizBodyProps): JSX.Element => {
```

Comment:
also i notice you export in this section but at the very bottom you still called export default, is there a reason for this?

Reply:
my thinking is that devs can import from the index as { AdminQuizBody }
or import from the file as just AdminQuizBody

Reply:
I'm not really sure about this approach yet though I will let you experiment on it.